.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
To make your test results easier to read, it's better to add `--format documentation` inside this file.

![test2](https://user-images.githubusercontent.com/31304943/70471158-f4be8a80-1acc-11ea-8fad-55a2f72dad78.png)

![test](https://user-images.githubusercontent.com/31304943/70471157-f425f400-1acc-11ea-9ed2-b85158a25efc.png)